src/types/index.ts

```diff
@@ -88,6 +88,7 @@ export type LocationStatus = 'notSet' | 'set' | 'unknown';
 export type StripeError<T = CommonError> = {
   code: T;
   message: string;
+  paymentIntent?: PaymentIntent.Type;
```

Comment:
it might be better to modify the `PaymentIntentResultType` and add a third option that has both error and PI populated